cogs/afterhours/afterhours.py

```diff
@@ -207,14 +208,42 @@ async def handleChannelDelete(self, channel: discord.abc.GuildChannel):
 
     @commands.group(name="afterhours")
     @commands.guild_only()
-    @checks.mod_or_permissions(manage_messages=True)
     async def afterHours(self, ctx: Context):
         """Configure after-hours exceptions
 
         There's nothing configurable from Discord.
         """
```

Comment:
I think it would be good to change this as well, maybe something along the lines of:
```suggestion
        """Manage after-hours"""
```